transfer-data/flows.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Flows'
+description: 'This page explains how to set up a flow to transfer and transform data from an Axiom dataset to a destination.'
+---
+
+import Prerequisites from "/snippets/minimal-prerequisites.mdx"
+
+<Note>
+Flow is currently in preview. To try it out, [sign up for a free preview](https://forms.axiom.co/flow).
+</Note>
+
+A flow is a way to transfer and transform data from an Axiom dataset to a destination that you choose. This page explains how to set up a flow.
+
+<Prerequisites />
+{/* list separator */}
+- Set up a destination. For more information, see [Destinations](/transfer-data/destinations).
+
+## Set up a flow
+
+To set up a flow:
+1. Click the **Flows** tab. Axiom displays the list of flows you have created.
+2. In the top right, click **New flow**.
+3. In the **Source** section, specify the source dataset and the transformation in an APL query. For example, the following APL query selects events from the `cloudflare-logpush` dataset and transforms the data by deleting and adding a field.
+
+    ```kusto
+    // Transfer high-value app events to a costly SIEM
+    // while retaining the complete record of all events in Axiom.
+    ['cloudflare-logpush']
+    | where QueryName == "app.axiom.co."
+    // Reduce events by dropping unimportant field
+    | project-away ['@app']*
+    // Enrich events with additional context
+    | extend ['@origin'] = "_axiom"
+    ```
+
+    <Note>
+    If you only specify the name of the dataset in the query, Axiom transfers all the raw data to the destination.
+    </Note>
+
+4. Click **Preview** to check whether the query you specified transforms your data as you want it. The **Input event** section displays the original data stored in Axiom. The **Output event** section displays the transformed data that Axiom sends to the destination. The raw data in Axiom isn’t affected by the transformation.
+5. In the **Destinations** section, click **Add destination**, and then select the destination where you want to transfer the transformed data. To add several destinations, click **+** in the top right corner.
+6. In the top right, click **Create**.
+
+GIF
+
+## Replay a flow
```

Comment:
I would like us to include a note that there are three kinds of flow: one-time, scheduled, and continuous, and that the preview will support one-time flows to begin with. This is critical for customers to understand.